src/modules/add-site/components/blueprints.tsx

```diff
@@ -267,40 +395,63 @@ export function AddSiteBlueprintSelector( {
 				{ __( 'Start from a blueprint' ) }
 			</Heading>
 
+			<BlueprintDetailsModal
+				warnings={ blueprintWarnings }
+				validationError={ validationError }
+				fileName={ selectedFileName || '' }
+				isOpen={ showWarningsModal }
```

Comment:
I think we can simplify the code with removing 1 state (`showWarningsModal`)
```suggestion
				isOpen={ !! blueprintWarnings }
```